manifests/machineconfig.crd.yaml

```diff
@@ -39,3 +39,183 @@ spec:
+                                type: integer
+                              name:
+                                description: "name is the group name of the owner"
+                                type: string
+                          overwrite:
+                            description: "overwrite specifies whether to delete preexisting nodes at the path"
+                            type: boolean
+                          append:
+                            description: "append specifies whether to append to the specified file. Creates a new file if nothing exists at the path. Cannot be set if overwrite is set to true."
+                            type: boolean
+                systemd:
+                  description: "systemd describes the desired state of the systemd units"
+                  type: object
+                  properties:
+                    units:
+                      description: "units is a list of units to be configured"
+                      type: array
+                      items:
+                        description: "items describes unit configuration"
+                        type: object
+                        properties:
+                          name:
+                            description: "name is the name of the unit. This must be suffixed with a valid unit type (e.g. 'thing.service')"
+                            type: string
+                          enabled:
+                            description: "enabled describes whether or not the service shall be enabled. When true, the service is enabled. When false, the service is disabled. When omitted, the service is unmodified. In order for this to have any effect, the unit must have an install section"
+                            type: boolean
+                          mask:
+                            description: "mask describes whether or not the service shall be masked. When true, the service is masked by symlinking it to /dev/null"
+                            type: boolean
+                          contents:
+                            description: "contents is the contents of the unit"
+                            type: string
+                          dropins:
+                            description: "dropins is the list of drop-ins for the unit"
+                            type: array
+                            items:
+                              description: "items describes unit dropin"
+                              type: object
+                              properties:
+                                contents:
+                                  description: "contents is the contents of the drop-in"
+                                  type: string
+                                name:
+                                  description: "name is the name of the drop-in. This must be suffixed with '.conf'."
+                                  type: string
+                kargs:
```

Comment:
kargs and fips are part of the MachineConfigSpec, not the ignition config (unless I'm messing up with indentation)